lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -152,14 +152,16 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
 
         if (!rootNodeId) {
             let nodeId: string;
```

Comment:
can we move what is inside this if in a separate method? try catch block should leave as good practice in a method and not find it in the middle of the code of your method. This is because try-catch is a concern and for the single responsibility principle any method should do only one thing

Reply:
Updated. Thanks